registry/client/repository_test.go

```diff
@@ -1019,12 +1029,12 @@ func TestManifestTagsPaginated(t *testing.T) {
 			"Last-Modified":  {time.Now().Add(-1 * time.Second).Format(time.ANSIC)},
 		})
 		if i < 2 {
-			headers.Set("Link", "<"+s.URL+"/v2/"+repo.Name()+"/tags/list?n=1&last="+tagsList[i]+`>; rel="next"`)
+			headers.Set("Link", "<"+s.URL+"/v2/"+repo.Name()+"/tags/"+ref+"?n=1&last="+tagsList[i]+`>; rel="next"`)
```

Comment:
This is split between different test cases. 1) is covered in `handlers/api_test.go` and 2) and 3) are covered in `client/repository_test.go`